pkgs/tools/graphics/pdf-sign/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+, substituteAll
+, ghostscript
+, pdftk
+, poppler_utils
+, coreutils
+
+, python3
+, makeWrapper
+}:
+
+let
+  python-env = python3.withPackages (ps: with ps; [ tkinter ]);
+in
+stdenv.mkDerivation {
+  pname = "pdf-sign";
+  version = "unstable-2023-08-08";
+
+  src = fetchFromGitHub {
+    owner = "svenssonaxel";
+    repo = "pdf-sign";
+    rev = "98742c6b12ebe2ca3ba375c695f43b52fe38b362";
+    hash = "sha256-5GRk0T1iLqmvWI8zvZE3OWEHPS0/zN/Ie9brjZiFpqc=";
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./use-nix-paths.patch;
+      gs = "${ghostscript}/bin/gs";
+      pdftk = "${pdftk}/bin/pdftk";
+      pdfinfo = "${poppler_utils}/bin/pdfinfo";
+      mv = "${coreutils}/bin/mv";
+    })
+  ];
+
+  buildInputs = [ python-env ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp pdf-sign pdf-create-empty $out/bin
+    patchShebangs $out/bin
+  '';
+
+  meta = {
```

Comment:
Please, keep as many attributes as possible sorted alphabetically, here the `meta` attributes needs to be sorted.

Reply:
I try to keep my attributes in a logical order, which might not be alphabetical.
I could put `description` to be first, sure, but having `maintainers` not be last is a bit weird IMO

Reply:
Mmh I can understand. Since there is no strict rule on this, we can be flexible. I see many people ordering them naturally, so that's the reason why I'm requesting contributors to do so.

I won't complain if you don't want to do it, but for the sake of consistency and reproducibility, I will ask you to think twice and do it 😅

Reply:
Alright